### ISSUE TITLE
Nested Array fields Editing auto collapsing Items

### ISSUE THREAD

**Issue by sagarchoudhary96** [measuredco/puck#200]
I have a config as 

```
"categories" : {
    "type": "array",
    "arrayFields": {
      "title": {
        "type": "text",
        "label": "Title"
      },
      "tabs": {
        "type": "array",
        "arrayFields": {
          "title": {
            "type": "text",
            "label": "Title"
          },
          "afterImageUrl": {
            "type": "text",
            "label": "AfterImage URL"
          },
          "beforeImageUrl": {
            "type": "text",
            "label": "BeforeImage URL"
          }
        }
      }
    }
  }
```

When i try to edit value of the fields of the `tabs` it collapses the item in tabs array on each keypress making it unusable to edit data.

@chrisvxd

**Comment by chrisvxd** [comment]
Thanks @sagarchoudhary96. Suspect this a bug introduced in #143.

**Comment by sagarchoudhary96** [comment]
yeah, this -> https://github.com/measuredco/puck/pull/143/files#diff-21349fb3247f4e87eab9004e83e0c2221019485c9ea5918b4e25569701d9b118R32-R39  seems to setting openId to "" on every render.

**Comment by chrisvxd** [comment]
@sagarchoudhary96 this should be fixed in `0.12.1-canary.15284aa`.
